src/main/java/com/vonage/quickstart/sms/SendSignedSms.java

```diff
@@ -38,14 +38,14 @@ public static void main(String[] args) throws Exception {
         String VONAGE_API_KEY = envVar("VONAGE_API_KEY");
         String VONAGE_SIGNATURE_SECRET = envVar("VONAGE_SIGNATURE_SECRET");
         String TO_NUMBER = envVar("TO_NUMBER");
-        String VONAGE_BRAND_NAME = envVar("VONAGE_BRAND_NAME");
+        String FROM_NUMBER = envVar("FROM_NUMBER");
```

Comment:
I think this should be a spec change, not a snippet change. We use `VONAGE_BRAND_NAME`  in the other SMS snippet, since it seems most other countries need an alpha-numeric sender (US is the oddball when it comes to forcing a number).

Reply:
fixed